jujugui/static/gui/src/app/app.js

```diff
@@ -760,13 +760,17 @@ YUI.add('juju-gui', function(Y) {
         how to render.
     */
     _renderInspector: function(metadata) {
+      var service = this.db.services.getById(metadata.id);
       React.render(
         <components.Panel
           instanceName="inspector-panel"
           visible={true}
           metadata={metadata}>
-          <components.Inspector>
-            <components.ServiceOverview />
+          <components.Inspector
+            service={service}
+            changeState={this.changeState.bind(this)}>
+            <components.ServiceOverview
```

Comment:
I feel like this should be determined inside the inspector component based on the state passed into the inspector. Because this body component will change based on the state.